docs/Intro_to_Medical_Image_Registration.ipynb

```diff
@@ -319,7 +319,7 @@
         "\n",
         "Combined with the regularisation on the predicted displacement field, this forms a weakly-supervised training. An illustration of a weakly-supervised DDF-based registration network is provided below.\n",
         "\n",
-        "When multiple labels are available for each image, the labels can be sampled during training, such that only one label per image is used in each iteration of the data set (epoch). Details are again provided in the [DeepReg dataset loader API](https://deepregnet.github.io/DeepReg/#/doc_data_loader) but not required for the tutorials.\n",
+        "When multiple labels are available for each image, the labels can be sampled during training, such that only one label per image is used in each iteration of the data set (epoch). Details are again provided in the [DeepReg dataset loader API](https://deepreg.readthedocs.io/en/latest/docs/dataset_loader.html) but not required for the tutorials.\n",
```

Comment:
Not sure if best practice, but could link to the requirements sub-section: https://deepreg.readthedocs.io/en/latest/docs/dataset_loader.html#dataset-requirements